interface/src/app/gene/genes.js

```diff
@@ -0,0 +1,376 @@
+angular.module('adage.gene.search', [
+  'adage.gene.resource'
+])
+
+.config(function($stateProvider) {
+  $stateProvider
+    .state( 'gene_search', {
+      url: '/gene_search',
+      views: {
+        "main": {
+          templateUrl: 'gene/gene-network.tpl.html',
+          controller: ['$scope', 'UserFactory',
+            function($scope, UserFactory) {
+              UserFactory.getPromise().$promise.then( function() {
+                $scope.userObj = UserFactory.getUser();
+              });
```

Comment:
Do we need to provide a error function to then() too, or it is not necessary here?